packages/cli/src/commands/upgrade.js

```diff
@@ -0,0 +1,50 @@
+import execa from 'execa'
+import Listr from 'listr'
+
+import c from 'src/lib/colors'
+
+export const command = 'upgrade'
+export const desc = 'Upgrade all @redwoodjs packages via interactive CLI'
+
+export const builder = (yargs) => {
+  yargs
+    .option('check', {
+      alias: 'c',
+      type: 'boolean',
+      default: false,
+      description: 'Check for outdated packages without upgrading',
+    })
+    .strict()
+}
```

Comment:
I think `--dry-run` is a common way of describing this scenario

Reply:
Ah, good call. Used on it’s own, the `yarn outdated` just check for outdated packages. But in this context it felt a little _off._ Making this change.